traffic_ops/traffic_ops_golang/logs/log.go

```diff
@@ -135,11 +163,64 @@ FROM "log" as l JOIN tm_user as u ON l.tm_user = u.id`
 
 const countQuery = `SELECT count(l.tm_user) FROM log as l`
 
+func getLogV40(inf *api.APIInfo, days int) ([]tc.Log, uint64, error) {
+	var count = uint64(0)
+	var whereCount string
+
+	queryParamsToQueryCols := map[string]dbhelpers.WhereColumnInfo{
+		"username": {Column: "u.username", Checker: nil},
+	}
+	where, _, pagination, queryValues, errs :=
+		dbhelpers.BuildWhereAndOrderByAndPagination(inf.Params, queryParamsToQueryCols)
+
+	if len(errs) > 0 {
+		return nil, 0, util.JoinErrs(errs)
+	}
+
+	timeInterval := fmt.Sprintf("l.last_updated > now() - INTERVAL '%v' DAY", days)
```

Comment:
nit but formatting that isn't meant to layout data structures (which should, itself, be rare) should not use `%v`, since a more specific formatting parameter would enforce a compile-time check on the type of the related argument

Reply:
updated.